examples/07-Train-an-xgboost-model-using-the-Merlin-Models-API.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>high-level API: "high-level" is used as an adjective and should be hyphenated.</p><p> </p><p>Instead of "currently", do you mind adding a time frame or a software version so that the sentence ages better?  Maybe "For the Merlin Models v0.6.0  release, some <code>xgboost</code> and <code>implicit</code> models are supported." </p><p> </p><p>nit: s/allows/enables/ -- I've been conditioned to avoid "allows" due to possible confusion related to granting permission.</p><p> </p><p>"try out" can be "evaluate" if you want a $10 word.  Up to you.  </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/522/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>